src/ScheduleService.Models.Dto/Responses/Group/DeleteGroupResponse.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace LT.DigitalOffice.ScheduleService.Models.Dto.Responses.Group;
+
+public class DeleteGroupResponse
+{
+  public bool Result { get; set; }
+
+  public string Erorr {  get; set; }
```

Comment:
It's true that there are three r's in this word, but you didn't guess the order

Reply:
+